### PR TITLE
update imagestream references

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[test][extended:core,cmd]

**Comment by bparees** [comment]
@smarterclayton ptal.  moving everything to the SCL image versions.

**Comment by bparees** [comment]
@liggitt can you take a look at this one?

**Comment by smarterclayton** [comment]
Looks ok.

**Comment by bparees** [comment]
[merge]

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -34,9 +34,9 @@ To create a new application, you can use the example app source. Login to your s
 run new-app:
 
   $ %[1]s login
-  $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
+  $ %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git
```
Comment (soltysh) [inline pkg/cmd/cli/cli.go]: Just out of curiosity, are we planning to update those regularly? I understand that for tests the answer is - yes,  unless we introduce global constants to simplify that process? Or this is just one time issue, again does not apply to tests.

**Comment by bparees** [comment]
[merge]

File: pkg/cmd/cli/cli.go (hunk 1 above)
Comment (bparees) [inline pkg/cmd/cli/cli.go]: they'll change each time we get a new version of ruby.  probably not that frequently.

Comment (bparees) [inline pkg/cmd/cli/cli.go]: and the existing text wasn't wrong, i just figured it would be nice for the example to point people to the current image.

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
[merge]

File: pkg/cmd/cli/cli.go (hunk 1 above)
Comment (soltysh) [inline pkg/cmd/cli/cli.go]: :+1:

**Pull request merged**